pkg/tnf/handlers/handler_template/handler.tmpl

```diff
@@ -36,61 +31,45 @@ const (
 	SuccessfulOutputRegex = `.+`
 )
 
-// Args returns the command line args for the test.
 func (h *{{ .Handlername }}) Args() []string {
-	return h.args
+	//fill the stub
 }
 
 // GetIdentifier returns the tnf.Test specific identifier.
 func (h *{{ .Handlername }}) GetIdentifier() identifier.Identifier {
-	return identifier.HostnameIdentifier
+	//fill the stub
 }
```

Comment:
can you update the comment to something like :
`// TODO create identifier.{{ .Handername}}Identifier; uncomment line below              

// return identifier.{{ .Handername}}Identifier
`